map/framework.cpp

```diff
@@ -2319,6 +2319,27 @@ bool Framework::ParseEditorDebugCommand(search::SearchParams const & params)
   return false;
 }
 
+namespace
+{
+vector<string> FilterNearbyStreets(vector<search::ReverseGeocoder::Street> const & streets)
```

Comment:
If you didn't get my response, it can be a sign that I did not get your question.

Am I right that you don't like this anonymous namespace function and want me to inline all it's code into several places where it is used now?